### PR TITLE
added token registration file for It's a pixel.

### PR DESCRIPTION
**Pull request by kylebehse**
# Pull Request Template

## Description

**Added a json file for the token registry of It's a pixel. token.**

## Type of change

- [ ] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

File: mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.itsapixel.com",
```
Comment (VladislavKudrin) [inline mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json:5]: Hi @kylebehse,

please put policyId to the website or remove the field.

Thanks!

Comment (kylebehse) [reply mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json:5]: Thanks @VladislavKudrin ,

I'm looking into what the requirements are for the policyId. The token will be used to support a larger company so I do not want any mention of it directly on the homepage but instead on a page like:
https://www.itsapixel.com/token-policy
I can update the url -> value to reflect that but was wondering if you had a template or any references to what is required to be on that page in order to meet this reqirement?

Thanks again,
Kyle

Comment (kylebehse) [reply mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json:5]: @VladislavKudrin ,

I couldn't find a requirements template so I just asked GPT. It gave me a bunch of suggestions, so I've started a document. Can you let me know if I'm on the right track?

[https://www.itsapixel.com/its-a-pixel-token-policy](url)

Note: It's just going to be used as a loyalty program. Like AirMiles that can be swapped on the market if someone doesn't want them. 

I'm just curious, if I continue with this template, will that meet the requirements?

Thanks so much for your time!

Comment (VladislavKudrin) [reply mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json:5]: @kylebehse,

the page should be accessible from the submitted URL. For example on https://www.itsapixel.com there is a link to token-policy page

Comment (kylebehse) [reply mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json:5]: @VladislavKudrin , ok great, thanks, I'll restructure and get back to you. Cheers!

**Comment by kylebehse** [comment]
@VladislavKudrin , sorry about the confusion. Updated url value to page with policyId and updated associated signature. Should be good to go??
Thanks so much!

**Comment by VladislavKudrin** [comment]
> @VladislavKudrin , sorry about the confusion. Updated url value to page with policyId and updated associated signature. Should be good to go?? Thanks so much!

@kylebehse,

all good, we can verify the submission! One thing though: you've put policy script cbor and not the policy id.

Here is your token's policy id: 86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd917
You can update it later to avoid any confusion.

Good luck!

**Review by VladislavKudrin — APPROVED** [review]

**Comment by kylebehse** [comment]
Updated! Thanks so much, @VladislavKudrin !!

**Pull request merged**